CONTRIBUTING.md

```diff
@@ -24,13 +24,13 @@ Before submitting large changes, please open an issue on GitHub outlining:
 - Detailed description of what your changes would entail.
 - Alternative solutions or approaches if applicable.
 
-Use your judgement about what constitutes a large change. If you aren't sure, send a message to the `#conftest` channel in the OPA slack or submit an issue on GitHub.
+Use your judgment about what constitutes a large change. If you aren't sure, send a message to the `#conftest` channel in the OPA slack or submit an issue on GitHub.
```

Comment:
Typo here, should be judgement right?

Reply:
LOL I actually Google'd this last night. Judgment _appears_ to be the "more correct" approach according to what I'm seeing, but in the end both are acceptable and didn't necessarily warrant a change. I just didn't bother going back and undoing it.